seeders/20200803000000-create tags.js

```diff
@@ -8,62 +8,62 @@ module.exports = {
         name: "Gluten",
         type: "allergen",
         excludeForFilter: true,
-      }, { transaction: t });
+      });
```

Comment:
Just to let you know that I removed the extraneous transaction. No problemo now.